src/Datadog.Trace.ClrProfiler.Native/environment_variables.h

```diff
@@ -97,6 +97,14 @@ namespace environment
     // The DOTNET_CLI_TELEMETRY_PROFILE in the context of azure app services
     const WSTRING azure_app_services_cli_telemetry_profile_value = WStr("DOTNET_CLI_TELEMETRY_PROFILE");
 
+    // The FUNCTIONS_WORKER_RUNTIME in the context of azure app services
+    // Used as a flag to determine that we are running within a functions app.
+    const WSTRING azure_app_services_functions_worker_runtime = WStr("FUNCTIONS_WORKER_RUNTIME");
+
+    // Determine whether to instrument within azure functions.
+    // Default is false until official support is announced.
+    const WSTRING azure_functions_enabled = WStr("DD_AZURE_FUNCTIONS_ENABLED");
```

Comment:
Shouldn't this be `DD_TRACE_AZURE_FUNCTIONS_ENABLED`?

Reply:
Sure, makes sense